dev/DataTarget/EnumFile.php

```diff
@@ -93,7 +93,7 @@ public function writeCases(SortingInterface $sorting): self
 
         $firstMethodPos = mb_strpos($enumContent, ' public ');
         $newEnumContent = mb_substr($enumContent, 0, $startEnum + 1);
-        $cases = array_unique($this->cases);
```

Comment:
This indeed looks like something that is currently broken, as array_unique calls the __toString method, which only relies on the name, not the value of an enum case;
```php
$array = [
    new EnumCase('foo', 'bar'),
    new EnumCase('foo', 'bop')
];

$array2 = array_unique($array); // [new EnumCase('foo', 'bar')]
```